index.js

```diff
@@ -15,6 +21,26 @@ app.get('/', (req, res) => {
   res.send('Hello World');
 });
 
+var master_products = all_products.products;
+master_products.forEach((each_product) => each_product.categoryName = all_categories.categories.filter(each_category => each_category.id == each_product.categoryId)[0].categoryName);
```

Comment:
Can you migrate this to a function and call it here? 
Or migrate to individual API methods.